src/photos/ducks/backup/hooks/useBackupActions.jsx

```diff
@@ -0,0 +1,149 @@
+import React, {
+  useState,
+  useEffect,
+  useCallback,
+  createContext,
+  useContext
+} from 'react'
+
+import { useWebviewIntent } from 'cozy-intent'
+
+import { useBackupData } from 'photos/ducks/backup/hooks/useBackupData'
+
+const BackupActionsContext = createContext(null)
+
+export const BackupActionsProvider = ({ children }) => {
+  const { setBackupInfo } = useBackupData()
+
+  const [backupPermissions, setBackupPermissions] = useState(null)
+  const [backupError, setBackupError] = useState(null)
+  const [isAllowPermissionsModalOpen, setIsAllowPermissionsModalOpen] =
+    useState(false)
+  const webviewIntent = useWebviewIntent()
+
+  const prepareBackup = useCallback(async () => {
+    try {
+      console.log('💿 📷 prepareBackup called')
```

Comment:
to remove? 

I think we have a logger in drive / photo that you can use, because it can be enabled / disabled globally.

Reply:
I removed them totally because they were not complete enough to be useful anymore. We will see if we need more logs later but I am not sure for the front end.